spf.go

```diff
@@ -196,6 +202,19 @@ func OverrideLookupLimit(limit uint) Option {
 	}
 }
 
+// OverrideVoidLookupLimit overrides the maximum number of void lookups allowed
+// during SPF evaluation. Note the RFC, SPF implementations SHOULD limit
+// "void lookups" to two.  An implementation MAY choose to make such a
+// limit configurable.  In this case, a default of two is RECOMMENDED.
+// Exceeding the limit produces a "permerror" result.
```

Comment:
Nit: I think this comment is correct and describe well what the RFC says, but it isn't necessarily the most useful thing for folks reading the documentation for this particular function, since it leaves a lot to be deduced by the reader.

How about something more direct, like:

```
OverrideVoidLookupLimit overrides the maximum number of void DNS lookups allowed
during SPF evaluation. A void DNS lookup is one that returns an empty answer, or a "name error".
Note that as per RFC, the default value of 2 SHOULD be used. Please use with care.
```